core/templates/components/question-difficulty-selector/question-difficulty-selector.component.css

```
@@ -0,0 +1,42 @@
+  margin-right: 1.2rem;
+  width: 9rem;
+}
+
+.oppia-skill-description-container {
+  font-weight: 400;
+  margin-left: 30px;
+  white-space: normal;
+  word-break: break-word;
+}
+
+/* :ng-deep is used to style the inner elements of the radio button.*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)